content/v1/deployment/helm/drivers/upgrade/powerstore.md

```diff
@@ -7,6 +7,10 @@ weight: 1
 Description: Upgrade PowerStore CSI driver
 ---
 
+{{% pageinfo color="primary" %}}
```

Comment:
Shouldn't we include all the file changes in v1? Won't we cherry pick all the file changes into the release-1.11 branch?